CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ All notable changes to this project will be documented in this file.
 -   #1750 : Add Python support for set method `remove()`.
 -   #1787 : Ensure `STC` is installed with Pyccel.
 -   #1743 : Add Python support for set method `discard()`.
+-   #1830 : Add a `pyccel.lambdify.lambdify` function to accelerate SymPy expressions.
```

Comment:
According to the quickstart file this function is just `pyccel.lambdify` without any repetition.

Reply:
Ah, this should read `pyccel.commands.lambdify.lambdify` because I moved the file. I wanted to reference the code but I did https://github.com/pyccel/pyccel/blob/f9753b6e87a815dd8d34d7280d3e7db65e5ab350/pyccel/__init__.py#L2

to allow the users to simply do `from pyccel import lambdify`. By the way I think we should do this for `epyccel` too. It always seems strange to write `from pyccel.epyccel import epyccel` rather than just `from pyccel import epyccel`. What do you think? I can open an issue if you want

Reply:
Yes, I agree! The user will only import a few things from Pyccel, so we should make them easy to find.

Reply:
https://github.com/pyccel/pyccel/issues/1836